relay-cabi/src/processing.rs

```diff
@@ -139,7 +139,6 @@ pub unsafe extern "C" fn relay_store_normalizer_normalize_event(
         is_renormalize: config.is_renormalize.unwrap_or(false),
         device_class_synthesis_config: false, // only supported in relay
         enrich_spans: false,
-        normalize_spans: false,
```

Comment:
Actually I'm not sure if this is a breaking change, going to verify it.

Reply:
I noticed we always run exclusive_time computation, even if the exclusive_time is already there. We should either add a check on field presence to prevent recomputation, or check the `is_renormalize` flag instead.

Reply:
I ran a bunch of tests related to the event manager with librelay in a local sentry instance and all passed in case I was missing something (although not required as we are explicitly setting the value there), so this should be fine.

Agree on the `is_renormalize` flag, implemented in [4741bf7](https://github.com/getsentry/relay/pull/3032/commits/4741bf790f27edaed20127c59d42ab2ad9efda53).